github/actions_secrets.go

```diff
@@ -89,30 +92,29 @@ func (s *ActionsService) GetSecret(ctx context.Context, owner, repo, name string
 	return secret, resp, nil
 }
 
+// EncryptedSecret represents represents a secret that is encrypted using public key.
+//
+// The value of EncryptedValue must value for your secret, encrypted with
```

Comment:
I'm having troubles parsing "The value of EncryptedValue must value for your secret". Can you please fix this comment?

Reply:
I change it slightly, is it better now?